apps/teams-test-app/index_cdn.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Teams Test App created using React"
+    />
+    <!--
+      manifest.json provides metadata used when your web app is installed on a
+      user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="./public/manifest.json" />
+    <title>Teams Test App</title>
+  </head>
+  <body>
+    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="https://res.cdn.office.net/teams-js/2.0.0-beta.1/js/MicrosoftTeams.min.js"
+      integrity="sha384-tQzNKNgoXVyO8/7UVmwwNjKy8Cai2COqlhP7/hL4Jz5flCpugVVTzNWo3SnGEpH4"
```

Comment:
Since the integrity hash will change for each build, we cannot hardcode the value here to validate the build, it will fail every time there is a change in SDK.  Lets add the dependency of webpack generating it https://www.npmjs.com/package/webpack-subresource-integrity(lets configure this) and get the integrity value dynamically for each build an use it in our tests.

Reply:
I like that, but that would mean we need to host the file locally and can't point at the CDN, because we won't be pushing to the CDN as part of every build, will we?

Reply:
We will be pushing for every build to CDN-SDF, and release version only to prod

Reply:
every PR build? Will that happen before automation would be invoked as part of the build itself?

Reply:
No not for every PR build but every merge to our release branch. No it wont happen before automation runs

Reply:
@MarcinJuraszek now I understood your concern(brain fade):
- Since this task is not intended to run on every build, and its purpose is to test the existing Prod content in CDN. WE DONT need to update the value for every build for this task(Spoke to @ella aswell about this).
- Though using the same setup(CDN test setup), we can run the tests after PROD/SDF CDN release, for those we might use the generate hash.
- We still need to update the integrity hash value in README.MD file for every build(separate from this task, will add a script to provision the vlaues generated by webpack-subresource-integrity dependency package)

Reply:
OK, I don't think I understand what this is trying to solve then.

Reply:
I don't think we want to update the integrity hash on every build. Only when we plan to publish a release to prod CDN.